app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,102 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if_logged_in do
+      @tweets = Tweet.all.dup
+      erb :'/tweets/index'
+    end
+  end
+
+
+  get '/tweets/new' do
+    if_logged_in do
+      erb :'/tweets/new'
+    end
+  end
+
+  post '/tweets' do
+    current_user.tap do |user|
+
+      if params[:content].chars.any?
```

Comment:
Whoa, thanks for mentioning that! I had basic validations in the User class, but I didn't realize you could do so much more with them!